packages/rocketchat-livechat/app/client/stylesheets/main.less

```diff
@@ -258,6 +258,13 @@ input:focus {
 					border-radius: 6px;
 					padding: 5px;
 					float: left;
+					word-wrap: break-word;
+					word-break: break-word;
+					-ms-word-break: break-all;
```

Comment:
@karlprieb is this the only concern on this PR? if so, we can edit it to use only `work-break` and merge it